### DISCUSSION TITLE
With some PRs the preview pane is not rendering fine anymore - how to track it down what breaks the UI?

### DISCUSSION THREAD

**Discussion by jgrumboe** [dlvhdr/gh-dash#492] [Q&A]
Hi,

With some PRs my preview pane is breaking the whole UI. Text bleeds into next line, menu and main list is completely messed up.
I somehow think it's related to PRs created by renovatebot.

Here is a screenshot with preview disabled, everything still working:
![image](https://github.com/user-attachments/assets/e93f7280-6cdc-4451-92ec-140437b0d72f)

And here I just press "p" to open the preview for the first PR - UI is broken:
![image](https://github.com/user-attachments/assets/cc837ec8-149d-4697-a2e1-21330f575b2a)

How can I hunt down what the problem could be to create a proper issue?

**Comment by dlvhdr** [comment]
Hmm what happens if you limit the preview sidebar size to a smaller or larger number?

**Reply by jgrumboe** [reply]
It get's more or less worse but doesn't go away. So, it seems to be related with the content length probably.
